crates/bitwarden-crypto/src/keys/master_key.rs

```diff
@@ -87,9 +87,14 @@ impl MasterKey {
 
     /// Decrypt the users user key
     pub fn decrypt_user_key(&self, user_key: EncString) -> Result<SymmetricCryptoKey> {
-        let stretched_key = stretch_kdf_key(&self.0)?;
+        let mut dec: Vec<u8> = match user_key {
+            EncString::AesCbc256_B64 { iv: _, data: _ } => user_key.decrypt_with_key(&self.0)?,
```

Comment:
Nit: I think this should also do the same:
```suggestion
            EncString::AesCbc256_B64 { .. } => user_key.decrypt_with_key(&self.0)?,
```

Reply:
This syntax eluded me for some reason.